library/Icinga/Application/Hook/PdfexportHook.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/* Icinga Web 2 | (c) 2018 Icinga Development Team | GPLv2+ */
+
+namespace Icinga\Application\Hook;
+
+/**
+ * Base class for the PDF Export Hook
+ */
+abstract class PdfexportHook
+{
+    /**
+     * Get whether PDF export is supported
+     *
+     * @return  bool
+     */
+    abstract public function isSupported();
```

Comment:
This could default to true, only implementations with third-party dependencies might want to override the default.

Reply:
This hook is for our chrome-headless pdfexport module in the first place and this one has deps.

Reply:
Others may not, so why force implementors to implement a useless method?

Reply:
If there's a pdfexport module other than our own on the horizon which does not have deps for rendering PDFs (good luck on that one), I'll happily change the method.